keyboards/idb_60/idb_60.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "quantum.h"
+
+inline void _idb_60_caps_led_on(void) {
+    setPinOutput(C5);
```

Comment:
You do not, and should not set the output with each on/off.  It only needs to be set on initialization.

```suggestion
```